.gitignore

```diff
@@ -7,6 +7,7 @@
 .lib/
 dist/*
 target/
+target3/
```

Comment:
We should find an alternative way to achieve this that follows sbt naming conventions. For synthetic JS/Native pure cross-projects they do `project.in(file(".js"))`.

Reply:
We need a new target directory if we want a new module in the same file, otherwise sbt doesn't compile. I don't think there is a naming convention for that.

Even if we just use the module for adding build target without making it publishable.